graphql/resolvers.go

```diff
@@ -1101,6 +1101,35 @@ func (r *taskResolver) PatchNumber(ctx context.Context, obj *restModel.APITask)
 	return &order, nil
 }
 
+func (r *taskResolver) CanRestart(ctx context.Context, obj *restModel.APITask) (bool, error) {
+	canRestart, err := canRestartTask(ctx, obj)
+	if err != nil {
+		return false, err
```

Comment:
making the canRestart field non-nullable means that we cannot return nil in the case of an error